modules/openy_features/openy_node/modules/openy_node_mbrshp/openy_node_mbrshp.features.yml

```diff
@@ -1 +1,3 @@
+excluded:
+  - core.entity_view_display.paragraph.membership_info.default
```

Comment:
Can we get rid of this? I don't see `core.entity_view_display.paragraph.membership_info.default` in the configs.